packages/components/src/internal/components/ontology/OntologyLookupOptions.tsx

```diff
@@ -66,7 +69,17 @@ export class OntologyLookupOptions extends PureComponent<Props, State> {
     };
 
     onFieldChange = (evt: any): void => {
-        this.onChange(evt.target.id, evt.target.value);
+        const val = evt.target.value;
+        let changes = List<IFieldChange>([{id: evt.target.id, value: evt.target.value }]);
+
+        const valFieldIdx = this.props.domainFields.findIndex(df => df.name === val);
```

Comment:
it looks like this `onFieldChange` is also called on change for the source ontology. In that case the `val` isn't a domain field name, so does that work as expected? I suppose it would result in a `-1` and skip the if statement below, right?

Reply:
Added a new handler for the Ontology selector just to be safe.